libraries/src/Extension/ExtensionHelper.php

```diff
@@ -208,6 +208,7 @@ class ExtensionHelper
         ['plugin', 'joomla', 'extension', 0],
         ['plugin', 'namespacemap', 'extension', 0],
         ['plugin', 'finder', 'extension', 0],
+        ['plugin', 'joomlaupdate', 'extension', 0],
```

Comment:
```suggestion
        ['plugin', 'joomla', 'extension', 0],
        ['plugin', 'joomlaupdate', 'extension', 0],
        ['plugin', 'namespacemap', 'extension', 0],
        ['plugin', 'finder', 'extension', 0],
```
Plugins of the same folder and client id should be alpha ordered. I know, the finder plugin should be moved, too, but I can't do this with a GitHub review suggestions as that would be too much above the code changed here.